app/models/case_request.rb

```diff
@@ -21,13 +21,15 @@ def initialize(opts = {})
   validate :case_must_exist_on_glimr, if: -> { errors.empty? }
 
   def process!
-    fee_liabilities.each do |fee|
-      prepare_fee(fee)
+    unless glimr_case_request.error?
```

Comment:
Wouldn't this have already been caught by the validation, or is it a mutant (or am I just being dumb)?